flink-runtime/src/test/java/org/apache/flink/runtime/io/disk/iomanager/IOManagerTest.java

```diff
@@ -61,12 +54,16 @@ public void channelEnumerator() throws Exception {
 
                 File path = id.getPathFile();
 
-                assertTrue("Channel IDs must name an absolute path.", path.isAbsolute());
-                assertFalse("Channel IDs must name a file, not a directory.", path.isDirectory());
+                assertThat(path.isAbsolute())
+                        .withFailMessage("Channel IDs must name an absolute path.")
+                        .isTrue();
+                assertThat(path.isDirectory())
+                        .withFailMessage("Channel IDs must name a file, not a directory.")
+                        .isFalse();
```

Comment:
```suggestion
                assertThat(path)
                        .withFailMessage("Channel IDs must name a file, not a directory.")
                        .isFile(();
```

Reply:
Thanks for catching this!

Reply:
The test will fail if we directly assert that path is file, because the file does not exist here.
So I created that file before asserting.